spyderlib/qt/__init__.py

```diff
@@ -57,6 +54,17 @@
             API_NAME += (" (API v%d)" % sip.getapi('QString'))
         except AttributeError:
             pass
+        from PyQt4 import uic  # analysis:ignore
```

Comment:
The `spyderlib.qt` package was wrote not only for Spyder but also for other libraries (`guidata`, `winpython` and now `python-qwt`). Adding the line `from PyQt4 import uic` here will simply allow to import the `uic` module with `from spyderlib.qt import uic`, that's all. FYI, the `uic` module is used to run GUI built with the help of QtDesigner (see [this example](https://github.com/PierreRaybaut/guiqwt/blob/master/guiqwt/qtdesigner.py)).